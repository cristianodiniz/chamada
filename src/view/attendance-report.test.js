import React from 'react';
import AttendanceReport from './attendance-report';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';

describe('Attendance Report Tests Component', function () {

  it('Should render without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<AttendanceReport />, div);
    ReactDOM.unmountComponentAtNode(div);
  });


  test('Should Match Snapshot', () => {
    const component = renderer.create(
      <AttendanceReport />,
    );
    
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });


})