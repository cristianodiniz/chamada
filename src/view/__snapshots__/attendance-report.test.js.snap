// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Attendance Report Tests Component Should Match Snapshot 1`] = `
<div
  className="MuiBox-root MuiBox-root-126"
>
  <header
    className="MuiPaper-root MuiPaper-elevation4 MuiAppBar-root MuiAppBar-positionFixed makeStyles-root-127 MuiAppBar-colorPrimary mui-fixed"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <button
        aria-label="Open drawer"
        className="MuiButtonBase-root MuiIconButton-root makeStyles-menuButton-128 MuiIconButton-colorInherit MuiIconButton-edgeStart"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiIconButton-label"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
            />
          </svg>
        </span>
      </button>
      <h6
        className="MuiTypography-root makeStyles-title-129 MuiTypography-h6 MuiTypography-noWrap"
      >
        Material-UI
      </h6>
      <div
        className="makeStyles-search-130"
      >
        <div
          className="makeStyles-searchIcon-131"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
            />
          </svg>
        </div>
        <div
          className="MuiInputBase-root makeStyles-inputRoot-132"
          onClick={[Function]}
        >
          <input
            aria-label="Search"
            className="MuiInputBase-input makeStyles-inputInput-133"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="Searchâ€¦"
            type="text"
          />
        </div>
      </div>
    </div>
  </header>
  <div
    className="makeStyles-space-134"
  />
  <ul
    className="MuiList-root ParticipantList-root-244 MuiList-padding"
  />
</div>
`;
